.github/ISSUE_TEMPLATE/release.md

```diff
@@ -18,32 +18,51 @@ Please assign the release manager to the issue.
 
 This issue can be closed when we have completed the following steps (in order).
 Please ensure all artifacts (PRs, workflow runs, Tweets, etc) are linked from
-this issue for posterity. Refer to this [prior release issue][release-1.11.0] for
+this issue for posterity. Refer to this [prior release issue][release-1.14.0] for
 examples of each step, assuming release vX.Y.0 is being cut.
 
-- [ ] **[In Crossplane Runtime]**: Prepared the release branch `release-X.Y` at the beginning of [Code Freeze]:
+### Code Freeze
```

Comment:
What about feature freeze? 🤔

Reply:
I don't think we have steps to perform on feature freeze day. We just stop accepting PRs with features.
Not sure if it is worth adding something here.